extensions/debuggee/src/org/exist/debuggee/ScriptRunner.java

```diff
@@ -69,7 +69,7 @@ public void run() {
 		try {
 			final Database db = BrokerPool.getInstance();
 			
-            BrokerPool.registerStatusObserver(this);
+//            BrokerPool.registerStatusObserver(this);
```

Comment:
@shabanovd I didn't say it was "thrown out", I said it was unsupported. AFAIK you added this code and since then no one has used it because the eXist-db website says "This is functional, but incomplete (the client side in particular)".